file_hook.go

```diff
@@ -54,6 +54,9 @@ func (hook *fileHook) Fire(entry *logrus.Entry) (err error) {
 			}
 		}
 		if hook.file == nil {
+			if err := os.MkdirAll(path.Dir(targetFile), 0766); err != nil {
```

Comment:
would this fail if the directories already exist?

Reply:
Nope, only creates dirs that don't exist

Reply:
Nope, only creates dirs that don't exist